frontend/src/Utilities/Date/timeZoneOptions.ts

```diff
@@ -0,0 +1,183 @@
+import { EnhancedSelectInputValue } from 'Components/Form/Select/EnhancedSelectInput';
+
+export const timeZoneOptions: EnhancedSelectInputValue<string>[] = [
+  { key: '', value: 'System Default' },
```

Comment:
`System Default` should be translated:

```suggestion
  {
    key: '',
    get value(): translate('SystemDefault'),
  },
```